scripts/save-crypto-prices-data.ts

```diff
@@ -0,0 +1,64 @@
+import ParsingClient from "sparql-http-client/ParsingClient";
+import axios from "axios";
+
+const SPARQL_ENDPOINT = process.env.SPARQL_ENDPOINT || "http://localhost:9999/blazegraph/sparql";
+const sparqlClient = new ParsingClient({ endpointUrl: SPARQL_ENDPOINT, updateUrl: SPARQL_ENDPOINT });
+
+const COIN_GECKO_API_URL = "https://api.coingecko.com/api/v3/";
+
+const getCryptocurrencyMarketData = async (name: string, startDate: number, endDate: number): Promise<any> => {
+    try {
+        const range = `range?vs_currency=usd&from=${startDate}&to=${endDate}`;
+        const { data, status } = await axios.get<any>(
+            `${COIN_GECKO_API_URL}/coins/${name}/market_chart/${range}`,
+            {
+                headers: {
+                    Accept: "application/json",
+                },
+            },
+        );
+
+        console.log(`Response status is: ${status}`);
```

Comment:
Do we still need this console.log()?

Reply:
I left it just to be sure that the call to the API was made successfully